PlaywrightEvershop/PageObjects/EverShopMenPage.cs

```diff
@@ -0,0 +1,54 @@
+﻿using Microsoft.Playwright;
+
+namespace PlaywrightEverShop.PageObjects;
+
+public class EverShopMenPage(IPage page)
```

Comment:
This page has method GoToMenPage(), so it can't be EverShopMenPage (it is also contains Women and Kids links at the top), it is EverShopMainPage or EverShopCatalogPage